### ISSUE TITLE
Use pictures everywhere

### ISSUE THREAD

**Issue by TrueHatori** [e107inc/e107#1522]
I like to use images that have already been uploaded, anywhere, for example, the photos from the news in a gallery, images from pages in the News and the gallery. But that's not because news pictures are not displayed in the Media Manager, if I want to put them in a gallery etc. Can you change that?

**Comment by Jimmi08** [comment]
If you use Media Manager, images are always displayed in media manager.  If they have category common images, they are always displayed. (If something didn't change from version 2.0) 
You can check in database if image is in media manager.

**Comment by TrueHatori** [comment]
The pictures are definitely in the database, they are uploaded with the Media Manager. In the Media Library they displayed. I tried "add to gallery", but when I then Media Categories visits and go to gallery, there on Image ... they are not there.

**Comment by Jimmi08** [comment]
In media manager find photo and check category, It's multicheck, you should add gallery category to display it in gallery, or (and) common images category to display it everywhere.

**Comment by TrueHatori** [comment]
Yes, I've made. I have also emptied the cache, the images are not displayed in the gallery selection. In v2.0 they were always all there, it was a bit tedious to find them, but they were there. Since v 2.1 it no longer works.
In the Media Libary both categories are shown for the pictures.

**Comment by Jimmi08** [comment]
If you choose just gallery, it works (just one value in multicheck)

this `_common_image,gallery_image `didn't go through regex:
`SELECT media_id FROM `e107_core_media` WHERE `media_category` REGEXP '(^|,)gallery_image(,|$)' AND `media_userclass` IN (253,254,250,251,0) ORDER BY media_id DESC `

**Comment by rica-carv** [comment]
I don't know if its related or not, but today i got a issue uploading media to the media manager, and after, associate it with the download images, and they weren't shown when i tried to selesct a image for a specific download...
I could only solve this but uploading images directly from the image picker when editing downloads...

By the way, a sugestion on image picker: why not use there a category filter??? If there's a lot of images, it gets a little bit tedious to load them all.....

**Comment by TrueHatori** [comment]
I just wanted to ask again if anyone has a solution. It can not be the meaning of that same image multiple upload, just because you want to use it in the news, the gallery and maybe even in a side :( Even for news and extended news you need it 2x.

**Comment by Moc** [comment]
Is this issue still occurring?

**Comment by Jimmi08** [comment]
https://github.com/e107inc/e107/issues/3905
